src/specmap/helpers.js

```diff
@@ -1,4 +1,7 @@
-export const freelyNamedKeyParents = [
+const freelyNamedKeyParents = [
+  'properties',
+]
+const freelyNamedStrParents = [
```

Comment:
We should probably be calling this `freelyNamedPaths` since these are complete paths, not parent keys (the original sin here is mine)

Reply:
Done ...
Also look at my question about consolidating some